rudder-web/src/main/resources/logback.xml

```diff
@@ -57,30 +57,31 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
    -->
 
 
+
   <!--
     Appender configuration - where&how to write logs in SLF4J speaking.
     ===================================================================
     Our default configuration : log on stdout appender so that our logs
     are managed by the container log system (and so, if Tomcat/Jetty/etc
     logs are stored in files and rotated, so are our log information).
-    
+
     Log format is:
     - date/time/thread of the log on 30 chars (fixed)
     - log level on 5 char (fixed)
     - name of the logger (and so the class) on 36 chars, with
       package name folding
     - log message follows
     - limit exception trace to 30 calls
-    
-    You should not have to modify that. 
+
+    You should not have to modify that.
   -->
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
       <Pattern>%d{[yyyy-MM-dd HH:mm:ss]} %-5level %logger - %msg%n%xEx{0}</Pattern>
```

Comment:
I know that it has nothing to do with the PR, but I'm questionning the utility to set stack trace to 0. If there is a problem, it may be interesting to have the trace. Pb with scala being that setting it 5 or something small generally don't help at all (it's likelly a bunch a map/flatmap calls...)